source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,130 @@
-Your First Task Script (NOT YET WRITTEN)
+Your First Task Script
 ----------------------
 
-**TODO: Write-up that guides the user through writing a basic task script (just a task with a narrate line) using VS-Code, emphasizing things like file extension and placement. Also run it in-game with `/ex run`.**
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+#### Task Script Basics
+
+A task script is a stand-alone script that can be run either via in-game command with the `/ex` [command](/guides/first-steps/ex-command), or by the `run` command in a script.
```

Comment:
Please put `/ex` inside the clickable bit, as that is part of the name of the section linked